Gemfile

```diff
@@ -0,0 +1,3 @@
+source "https://rubygems.org"
```

Comment:
Because `Gemfile.lock` is a much better way of being sure you get the same library versions when you setup a dev env than a `gemspec`, while continuing to allow better flexibility for how the `gemspec` specifies dependencies.